### PR TITLE
[WIPTEST] moving Host to navmazing lib

### PR DESCRIPTION
**Pull request by izapolsk**
Purpose or Intent
=================
With accordance to our plans to move all navigation to navmazing lib, I've moved Infrastructure/Host obj to the new navigation approach.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 6c9fde91cf4cd8f35406c4bc830d1bb012857bda:

`cfme/infrastructure/host.py`:
- :red_circle: Line 15:1: **F401** *'nav' imported but unused*
- :red_circle: Line 403:9: **F811** *redefinition of unused 'nav' from line 15*

`cfme/tests/configure/test_visual_infra.py`:
- :red_circle: Line 13:1: **F811** *redefinition of unused 'navigate_to' from line 6*

`cfme/tests/infrastructure/test_host_provisioning.py`:
- :red_circle: Line 4:1: **F401** *'sel' imported but unused*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f327c4e3f5cdd0148e018df9a6f40ea7bc059e57:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit cafda3da4bd709427c4b6ac0859f4b73e79bcc64:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/infrastructure/host.py [hunk 1]
```diff
@@ -140,6 +123,7 @@ def __init__(self, name=None, hostname=None, ip_address=None, custom_ident=None,
                  host_platform=None, ipmi_address=None, mac_address=None, credentials=None,
```
Comment (psav) [inline cfme/infrastructure/host.py]: For now we still need the ```self.appliance = ``` line in the ```__init__```

File: cfme/infrastructure/host.py [hunk 2]
```diff
@@ -408,15 +386,84 @@ def equal_drift_results(self, row_text, section, *indexes):
...
+    def step(self):
+        cfg_btn('Edit this item')
+
+
+@navigator.register(Host, 'Add')
+class Add(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        cfg_btn('Add a New item')
```
Comment (psav) [inline cfme/infrastructure/host.py]: Thanks for taking out the old version picks

Comment (izapolsk) [inline cfme/infrastructure/host.py]: I guess we'll need to do that everywhere.

File: cfme/infrastructure/host.py (hunk 1 above)
Comment (izapolsk) [inline cfme/infrastructure/host.py]: Fixed. I thought class attribute would be enough. I haven't heard about hot change of appliance in our framework.

**Comment by dajoRH** [comment]
Lint report for commit 6c997ed9f0a5f55907c62535e760afc265684d17:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/infrastructure/host.py (hunk 1 above)
Comment (psav) [inline cfme/infrastructure/host.py]: Nope

File: cfme/infrastructure/host.py (hunk 2 above)
Comment (psav) [inline cfme/infrastructure/host.py]: Yeh slowly

**Comment by dajoRH** [comment]
Lint report for commit 6ac7a7052567bc098ba70ed001c4621d7ec80039:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit c349e5fd7697b28866ffe2507dfd4cb62c1db1ed:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Pull request merged**